drafts/draft-ietf-ippm-ioam-direct-export.xml

```diff
@@ -223,10 +224,10 @@
 	  
 	  <t>This document defines a new IOAM option type (also known as an IOAM 
 	  type) called the Direct Export (DEX) option. This option is used as a 
-	  trigger for IOAM nodes to export IOAM data to a receiving entity (or 
-	  entities). A "receiving entity" in this context can be, for example, an 
-	  external collector, analyzer, controller, decapsulating node, or a 
-	  software module in one of the IOAM nodes.</t>
+	  trigger for IOAM nodes to locally aggregate IOAM data, or to export it 
```

Comment:
How about "..,to locally aggregate or process IOAM data..." to underline that the node might do all the processing itself and not send any additional packet(s).

It might even make sense to add a dedicated sentence that underlines this fact. E.g.
"Note: Even though the IOAM Option-Type is called "Direct Export", it depends on the deployment, whether the receipt of a packet with DEX Option-Type leads to the creation of another packet. Some deployments might simply use the packet with DEX Option-Type to trigger local processing of OAM data".